bedrock/foundation/templates/foundation/annualreport/2011.html

```diff
@@ -79,9 +79,9 @@ <h2 class="section-title">Mozilla Mobilized</h2>
       <section class="sub-section mob-intro">
         <div class="overlay-wrap">
           <ul class="mzp-c-inline-list">
-            <li id="topic-firefox"><img src="{{ static('img/foundation/annualreport/2011/logo-firefox.png') }}" alt="">Firefox</li>
-            <li id="topic-firefoxos"><img src="{{ static('img/foundation/annualreport/2011/logo-firefoxos.png') }}" alt="">Firefox OS</li>
-            <li id="topic-webmaker">{{ high_res_img('img/foundation/annualreport/2011/logo-webmaker.png', {'alt': ''}) }}Webmaker</li>
+            <li id="topic-firefox"><img src="{{ static('img/foundation/annualreport/2011/logo-firefox.png') }}" alt="" width="162" height="162">Firefox</li>
+            <li id="topic-firefoxos"><img src="{{ static('img/foundation/annualreport/2011/logo-firefoxos.png') }}" alt="" width="200" height="185">Firefox OS</li>
+            <li id="topic-webmaker">{{ high_res_img('img/foundation/annualreport/2011/logo-webmaker.png', {'alt': '', 'width': '170', 'height': '170'}) }}Webmaker</li>
```

Comment:
Not necessary, but wondering if we should remove the `high_res_img` here to use `resp_img` since high_res_img is depreciated. Up to you!

Reply:
I think I'm going to do a separate search for macros missing width, so I'll leave it alone for now.